homeassistant/helpers/device_registry.py

```diff
@@ -788,6 +793,8 @@ def async_update_device(
         add_config_entry_id: str | UndefinedType = UNDEFINED,
         area_id: str | None | UndefinedType = UNDEFINED,
         configuration_url: str | URL | None | UndefinedType = UNDEFINED,
+        device_info_type: str | UndefinedType = UNDEFINED,
```

Comment:
Only semi-related and maybe overkill since there are only 3 possible values now but might be a nice future cleanup to make `_validate_device_info` return a StrEnum / make `device_info_type` a StrNum.